biolink-model.yaml

```diff
@@ -122,6 +122,10 @@ slots:
     inherited: true
     mappings:
       - SEMMEDDB:ASSOCIATED_WITH
+      - BFO:0000054
+      - core#protects
```

Comment:
@cbizon What is this predicate?

Reply:
Oh, thanks for pointing that out.  All of these things that say core#whatever should be replaced with UBERON_CORE:whatever, where UBERON_CORE expands to "http://purl.obolibrary.org/obo/uberon/core#".  @YaphetKG is going to clean this up today.

Reply:
I've updated the curies accordingly, thanks @deepakunni3